common/production_methods/imperia_urban.txt

```diff
@@ -127,16 +127,16 @@ pm_motor_buses = {
 	building_modifiers = {
 		workforce_scaled = {
 			goods_input_automobiles_add = 2
-			goods_input_oil_add = 5
+			goods_input_oil_add = 2
 			# output goods
-			goods_output_transportation_add = 25
+			goods_output_transportation_add = 20
 			
-			#profit = (25*30)-((2*100)+(5*40)) = 350*52 = 18,200
+			#profit = (20*30)-((2*100)+(2*40)) = 320*52 = 16640
 		}
 
 		level_scaled = {
-			building_employment_laborers_add = 1000
-			building_employment_engineers_add = 500
+			building_employment_machinists_add = 600
+			building_employment_engineers_add = 600
```

Comment:
Why are we eliminating all laborers? I think the structure of these employment levels need to be fixed. You should be slightly decreasing laborer count each level, but not 1-1 converting laborers into engineers and such.